wire-ios-sync-engine/Source/SessionManager/SessionManager+Backup.swift

```diff
@@ -56,9 +55,17 @@ extension SessionManager {
             applicationContainer: sharedContainerURL,
             dispatchGroup: dispatchGroup,
             databaseKey: activeUserSession.managedObjectContext.databaseKey,
-            completion: { [dispatchGroup] in
+            completion: { [dispatchGroup] result in
+                switch result {
+                case .success:
+                    break
+                case .failure:
+                    let exportBackupFailed = ExportBackupFailed()
+                    activeUserSession.analyticsSession?.trackEvent(exportBackupFailed)
```

Comment:
Perhaps this can be handled in the method below? Also, would we have a session available at this time to track the event?

Reply:
This is an open question. I can give it a go and report back.